### ISSUE TITLE
Make it possible to format Clojure code using the pretty printer

### ISSUE THREAD

**Issue by PEZ** [BetterThanTomorrow/calva#1843]
Questions like this show up now and then: https://ask.clojure.org/index.php/12126/there-way-place-elements-collection-vertically-with-calva

TL;DR: Can Calva break up forms, adding new lines?

It can't. A crude (and often insufficient) workaround is to evaluate the form and get it pretty printed.

However, one of Calva's pretty printing options is built in to Calva and is happening client side (using zprint). We could use this to help format code/forms without evaluating it.

I see two main ways we could do this:

1. Make it possible to use zprint in addition to cljfmt as the Calva formatter
2. Add a command or two for pretty print-formatting the current form/selection/file

The former has been my ”plan” for very long, but it is not super easy to achieve. In fact it seems pretty hard to do it. The latter is probably not very hard. We have all the needed components, and just need to compose them a bit.
